jetpack-start/wp-cli-api.php

```diff
@@ -79,23 +79,35 @@ public function connect( $args, $assoc_args ) {
 			// Instead of repeatedly calling restore_current_blog() just to switch again, manually switch back at the end
 			$starting_blog_id = get_current_blog_id();
 
+			// Track whether there were any failures, to adjust messaging
+			$success = true;
+
 			foreach ( $sites as $site ) {
 				switch_to_blog( $site );
 
 				WP_CLI::line( sprintf( 'Starting %s (site %d)', home_url( '/' ), $site ) );
 
-				$this->connect_site( $assoc_args );
+				$_success = $this->connect_site( $assoc_args );
+
+				// Just one failure to flip the message from success
+				if ( true === $success && $success !== $_success ) {
```

Comment:
Minor: We can just simplify this as:

```
if ( true !== $_success ) {
    $success = $_success;
}
```

Since the cost of assignment is not that high.

For clarity, I would also rename `$success` to something more verbose like `$did_any_fail` (although, that flips the boolean and would require changes below).